### ISSUE TITLE
Documentation lacks more examples

### ISSUE THREAD

**Issue by andredias** [merll/docker-fabric#4]
I'd like very much to use docker-fabric in my projects (especially in testing) but I'm having a hard time trying to figure out how to use it. The examples in documentation are too basic and shallow to be useful. There are no unit tests and I couldn't find any projects or examples elsewhere using docker-fabric. I wonder if you could take some examples from fabric's docs and use them in a docker-fabric's context.

I miss an example that covers something from beginning to end such as:
1. build a docker image
2. create and start a container
3. run some commands in the container and several other tasks
4. stop and destroy the container

Regards,

André

**Comment by merll** [comment]
I agree that the structure of the current documentation is not optimal especially for the start. "[Getting started](http://docker-fabric.readthedocs.org/en/stable/start.html)" only covers the essentials to see how Docker API connections work through Fabric's SSH tunnel.
However, in addition to building Dockerfiles remotely, the main benefit comes from [reusable container configurations in Docker-Map](http://docker-fabric.readthedocs.org/en/stable/guide/containers.html). While the documentation of Docker-Fabric is not supposed to be too redundant to the more general Docker-Map library, there should be at least a practical example, covering the whole lifecycle of containers.

In fact there have been similar suggestions in a [PR to Docker-Map](https://github.com/merll/docker-map/pull/10), and the developer has published a [good use case for a WordPress setup](https://github.com/hsoft/wordpress-template-docker). It could be adapted to Docker-Fabric easily, so I will look into that.

**Comment by rodrigorodriguescosta** [comment]
The same here, I can't use, I just want to run a container, it would be nice if the project has examples with code to execute the steps above, I guess it would be more useful than the doc.. :(
